hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -394,6 +394,18 @@ class HassioAddonInfo extends LitElement {
                     haptic
                   ></ha-switch>
                 </div>
+                ${this.hass.userData?.showAdvanced
+                  ? html`
+                      <div class="state">
+                        <div>Watchdog</div>
```

Comment:
Should we add some explaination? Like: This will restart the add-on if it crashes.

Reply:
Yes, I have planned to to a  change to all these settings later this week (buy utilizing the ha-settings-row), so will add that then (with some description for the rest of the options as well.